plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeMinioConnectorTest.java

```diff
@@ -512,6 +517,27 @@ public void testPathColumn()
         }
     }
 
+    @Test
+    public void testTableWithNonNullableColumns()
+    {
+        String tableName = "test_table_with_non_nullable_columns_" + randomTableSuffix();
+        assertUpdate("CREATE TABLE " + tableName + "(col1 INTEGER NOT NULL, col2 INTEGER)");
+        assertUpdate("INSERT INTO " + tableName + " VALUES(1, 10)", 1);
+        assertThatThrownBy(() -> query("INSERT INTO " + tableName + " VALUES(null, 1)"))
+                .hasMessageContaining("NULL value not allowed for NOT NULL column: col1");
+
+        //TODO https://github.com/trinodb/trino/issues/13434
+//      assertThatThrownBy(() -> query("INSERT INTO " + tableName + " VALUES(TRY(5/0), 1)"))
+//              .hasMessageContaining("NULL value not allowed for NOT NULL column: col1");
```

Comment:
Can you please uncomment the assertion and let the query succeed?
and add a TODO explaining what the expected result should be?

this would ensure this code gets updated when the bug is fixed